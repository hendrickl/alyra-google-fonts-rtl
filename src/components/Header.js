import React from "react"

const Header = () => {
  return (
    <header className="p-5 bg-danger text-white text-center">
      <h1 className="display-1">Quoi de neuf, Google Fonts ?</h1>
    </header>
  )
}

export default Header
