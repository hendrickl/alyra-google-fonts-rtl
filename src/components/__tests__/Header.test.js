
import React from "react"
import Header from "../Header"
import { render, cleanup } from '@testing-library/react'

afterEach(cleanup)


test('<Header />', () => {
  const {
    debug, container,
  } = render(<Header />);

  expect(container.firstChild).toMatchSnapshot()
})
