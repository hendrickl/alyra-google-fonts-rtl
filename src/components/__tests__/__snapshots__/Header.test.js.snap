// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> 1`] = `
<header
  class="p-5 bg-danger text-white text-center"
>
  <h1
    class="display-1"
  >
    Quoi de neuf, Google Fonts™ ?
  </h1>
  <p
    class="h4"
  >
    Le plus récentes, les plus 
    <i>
      trendy
    </i>
     et les plus populaires polices Google.
  </p>
</header>
`;
